colossalai/shardformer/policies/chatglm2.py

```diff
@@ -168,22 +184,33 @@ def module_policy(self) -> Dict[Union[str, nn.Module], ModulePolicyDescription]:
                 target_key="ChatGLMModel",
             )
 
-        # use flash attention
-        if self.shard_config.enable_flash_attention:
+        # use sequence parallel
+        if self.shard_config.enable_sequence_parallelism:
```

Comment:
I think we don't need to replace it when using pipeline

Reply:
It is caused by the git diff algorithm. I have pushed a new commit to fix it.

Reply:
Added an if statement.